packages/time-utilities/src/lib/constants.ts

```diff
@@ -0,0 +1,74 @@
+export const enum Time {
+	Millisecond = 1,
+	Second = 1000,
+	Minute = 1000 * 60,
+	Hour = 1000 * 60 * 60,
+	Day = 1000 * 60 * 60 * 24,
+	Month = 1000 * 60 * 60 * 24 * (365 / 12),
+	Year = 1000 * 60 * 60 * 24 * 365
+}
+
+export const days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
```

Comment:
What kind of sin is this, Sunday as the first day???

Reply:
Welcome to USA's system, lol.